prime-router/src/test/kotlin/fhirengine/engine/FhirTranslatorTests.kt

```diff
@@ -106,10 +112,11 @@ class FhirTranslatorTests {
         val actionLogger = mockk<ActionLogger>()
         val engine = makeFhirEngine()
 
+        val reportId = UUID.randomUUID()
         val message =
             spyk(
                 FhirTranslateQueueMessage(
-                    UUID.randomUUID(),
+                    reportId,
```

Comment:
I think what happened here is we coded a bug and then got the tests to pass with the bug! This is one of the problems with our "unit tests". They require so much mocking that at some point it becomes hard to understand what we are even testing and if its correct. Thus, a lot of our Universal Pipeline steps have integration tests that do not mock and its easier to test things like this.

That said, my recommendation is to update the tests in `FHIRTranslatorIntegrationTests.kt` in addition to these.

Reply:
Are you saying that changing it to reportId from random UUID is the correct way to go?